builder/static_code.go

```diff
@@ -888,41 +951,208 @@ func listJoin(list []string, sep string, lastSep string) string {
 	}
 }
 
-func (p *parser) parseRule(rule *rule) (any, bool) {
+// ==template== {{ if .LeftRecursion }}
+
+func (p *parser) checkPrevChoice(rule *rule) (bool, bool) {
+	var currentEStack []any
+	var prevEStack []any
+	for i := 1; i <= len(p.rstack); i++ {
+		indexCurrent := len(p.rstack) - i
+		indexPrev := len(p.rstack) - i*2
+		if indexPrev < 0 {
+			continue
```

Comment:
Shouldn't this be `break` instead of `continue`?

Reply:
Fixed